internal/nodes/avi_model_l7_hostname_shard.go

```diff
@@ -358,9 +358,8 @@ func sniNodeHostName(routeIgrObj RouteIngressModel, tlssetting TlsSettings, ingN
 				certsBuilt = true
 			}
 		}
-		if lib.GetSEGName() != lib.DEFAULT_SE_GROUP {
-			sniNode.ServiceEngineGroup = lib.GetSEGName()
-		}
+
+		sniNode.ServiceEngineGroup = lib.GetSEGName()
```

Comment:
Can you also check for same block of code for EVH? Rebase, as the code is merged yesterday.